facturacion_mexico/ereceipts/doctype/ereceipt_mx/ereceipt_mx.py

```
@@ -71,201 +59,82 @@ def get_default_expiry_days(self):
 
-		# Cancelar en FacturAPI si existe
 		if self.facturapi_id:
 			try:
-				from facturacion_mexico.facturacion_fiscal.api_client import FacturapiClient
+				from facturacion_mexico.facturacion_fiscal.api_client import get_facturapi_client
 
-				client = FacturapiClient()
-				client.cancel_receipt(self.facturapi_id, reason)
+				client = get_facturapi_client(company=self.company)
+				client.cancel_receipt(self.facturapi_id)
 			except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)